connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointTask.java

```diff
@@ -132,7 +132,7 @@ public String version() {
             return listConsumerGroupOffsets(group).entrySet().stream()
                 .filter(x -> shouldCheckpointTopic(x.getKey().topic()))
                 .map(x -> checkpoint(group, x.getKey(), x.getValue()))
-                .filter(x -> x.downstreamOffset() > 0)  // ignore offsets we cannot translate accurately
+                .filter(x -> x.downstreamOffset() >= 0)  // ignore offsets we cannot translate accurately
```

Comment:
Can downstream offset be negative?

Reply:
nope, that's not possible

Reply:
Then, this filter matches everything, doesn't it?

Reply:
@ryannedolan @heritamas If you guys think this is safe enough I can remove that filter. But it doesn't hurt to leave it there just in case a rogue negative offset comes through for whatever reason/bug... Please let me know what you think.

Reply:
Yes, does not hurt to leave it. Just for sure.